nbs/explains/minidataapi.ipynb

```diff
@@ -0,0 +1,516 @@
+{
```

Comment:
_Commented on notebook `nbs/explains/minidataapi.ipynb` Cell 5 Line 1_
```python
db = database('data/minidataapi.db')
```
maybe use in-memory for this nb?